### DISCUSSION TITLE
permission denied

### DISCUSSION THREAD

**Discussion by samene** [Altinity/clickhouse-backup#489] [Q&A] [answered]
I am getting this error while doing a restore

`
can't attach partitions for table 'rb_mdm.dim_ixengine_service': code: 1001, message: std::__1::__fs::filesystem::filesystem_error: filesystem error: in directory_iterator::directory_iterator(...): Permission denied [/var/lib/clickhouse/store/a8b/a8b3c33b-a9b8-4780-a8b3-c33ba9b8f780/detached/attaching_all_0_0_0/]
`

I am running clickhouse in kubernetes (not using the altinity operator) pod as user 'clickhouse' uid 101 and all /var/lib/clickhouse is full read/write to this user and mounted as a volume to host. What am I doing wrong?

**Comment by Slach** [comment]
How do you run clickhouse-backup?
Do you run clickhouse-backup container in the same pod with clickhouse-server?
Do you use API or cli interface?
Do you use `restore_remote` or `download` + `restore` command?

**Reply by samene** [reply]
Ok I was able to resolve this. The backup container was running as root and the clickhouse container as 101. After running both containers as same user the issue was resolved.

**Comment by Slach** [comment, marked as answer]
> The backup container was running as root and the clickhouse container as 101. After running both containers as same user the issue was resolved.

great
